app/models/spree/user.rb

```diff
@@ -1,7 +1,7 @@
 module Spree
   class User < Spree::Base
+    include Spree::UserMethods
```

Comment:
@krzysiek1507 `UserMethods` will work, no need to use Spree namespace as we are already in the namespace